tests/Tests/500__jupyterhub/test.robot

```diff
@@ -74,7 +74,30 @@ Can Launch Python3
    [Tags]  Sanity  TBC
    Launch Python3 JupyterHub
 
+Verify Failing Images Are Shown In Notifications
+    [Documentation]    Verifies failing Notebook names are shown in RHODS Dashboard Notifications
+    [Tags]    Sanity
+    ...       ODS-470
+    ${build_name} =    Search Last Build    namespace=redhat-ods-applications    build_name_includes=pytorch
+    Delete Build    redhat-ods-applications    ${build_name}
```

Comment:
deleting the build doesn't lead to a "failed" build message in the dashboard

Reply:
Updated